packages/frontend/web-ui/src/game/helpers/getImageCard.ts

```diff
@@ -0,0 +1,25 @@
+export type KindCard = 'draw' | 'wild' | 'reverse' | 'skip' | 'wildDraw4';
+
+export function getImageCard(kind: KindCard): string {
```

Comment:
```suggestion
export function getImageCardUrl(card: Card): string {
```

I don't think adding a `CardKind` type is the best way to go. the API model `Card` is the source of truth here, I think everything is simpler if we rely on the API model instead of adding a type that needs to be sync with the `Card` model whenever it changes.